### ISSUE TITLE
issue while outputing to csv

### ISSUE THREAD

**Issue by giriyadigu** [routetehpacketz/cisco-ip-trace#5]
Dear sir,

The script is giving error if I outputting to csv or anything. 
If I run without output it is working fine. This is for your info. 
If you needed any further checks and log report. Pls let me know.
Thanks in advance.
Regards,

**Comment by routetehpacketz** [comment]
Issue has been corrected. I just forgot to remove the `options.` in front of `network_to_scan` in the `elif` statement used when an output file name is provided. Can you please confirm it is working for you now?

**Comment by giriyadigu** [comment]
Hello sir, yes now it is working. I can output the file into csv. Thanks for your immediate support.

**Comment by giriyadigu** [comment]
where can i find guide to make necessary changes as in our network we have port-channel configured and edge switch and ports are not identifying.

the output files is as below
----------------------------------------------------------------------------------------------------------
Tracing 10.80.2.15...Port Unknown
Device IP,MAC Address,Switch,Port,Port Description,Interface Type,VLANs on port,Port MAC count
10.80.2.15,14b3.1f0f.f9ec,INAX01-VSS,Unknown


Tracing 10.80.2.16...complete!

Device IP,MAC Address,Switch,Port,Port Description,Interface Type,VLANs on port,Port MAC count
10.80.2.16,e41f.1336.c070,INAX01-VSS,Gi2/1/44,KRSingh_LAB,access,"5",8


Tracing 10.80.2.17...Port Unknown
Device IP,MAC Address,Switch,Port,Port Description,Interface Type,VLANs on port,Port MAC count
10.80.2.17,14b3.1f10.2f5e,INAX01-VSS,Unknown


Tracing 10.80.2.18...MAC not found in ARP
Device IP,MAC Address,Switch,Port,Port Description,Interface Type,VLANs on port,Port MAC count
10.80.2.18,Not Found


Tracing 10.80.2.19...Port Unknown
Device IP,MAC Address,Switch,Port,Port Description,Interface Type,VLANs on port,Port MAC count
10.80.2.19,4437.e64b.d6e8,INAX01-VSS,Unknown)

**Comment by giriyadigu** [comment]
Dear sir,

Tested found ok, Now it is working fine, I am able to output the file to
*.CSV.

But being a non-programming knowledge, I am not able to guess why I am
still not getting the full results.
only directly connected ports on the switch is showing the full results, if
it is connected to an access switch the result or failing
as we have configured VSS... Port channel configuration.

----------------------------------------------------------------------------------------------------------------------------------------------------
Device IP MAC Address Switch Port Port Description Interface Type VLANs on
port Port MAC count
10.80.2.1 004e.01b5.ebe9 INAX01-VSS Unknown



10.80.2.2 Not Found





10.80.2.3 0023.24a9.62dd INAX01-VSS Unknown



10.80.2.4 14b3.1f10.2f85 INAX01-VSS Unknown



10.80.2.5 6cf0.492a.eb22 INAX01-VSS Unknown



10.80.2.6 Not Found





10.80.2.7 004e.01b5.ed4e INAX01-VSS Unknown



10.80.2.8 004e.01b5.ed1b INAX01-VSS Unknown



10.80.2.9 14b3.1f1e.2f87 INAX01-VSS Unknown



10.80.2.10 0010.7535.96fa INAX01-VSS Gi1/1/16 Production Storagebox access 5
4
10.80.2.11 Not Found





10.80.2.12 14b3.1f0f.fa12 INAX01-VSS Unknown



10.80.2.13 b885.84c1.f606 INAX01-VSS Unknown



10.80.2.14 004e.01b5.ed53 INAX01-VSS Unknown



10.80.2.15 14b3.1f0f.f9ec INAX01-VSS Unknown



10.80.2.16 e41f.1336.c070 INAX01-VSS Gi2/1/44 KRSingh_LAB access 5 8
10.80.2.17 14b3.1f10.2f5e INAX01-VSS Unknown



10.80.2.18 Not Found





10.80.2.19 4437.e64b.d6e8 INAX01-VSS Unknown



----------------------------------------------------------------------------------------------------------------------------------------------------------------

your valuable  guidance will be very much appreciated. Thanks in advance.

Regards,

Girish
